vignettes/algorithm.Rmd

```diff
@@ -98,6 +98,36 @@ where $\Omega \in \mathbb{R}^{N \times N}$ is block-diagonal containing the
 subject specific $\Sigma_i$ covariance matrices on the diagonal and 0 in the
 remaining entries.
 
+### Linear model with group covariance matrix
```

Comment:
thanks @clarkliming this is a good start and you understood the problem well! To simplify the vignette, I would recommend to 
1) move this as a subsubsection within the `Covariance matrix model` subsection
2) simplify the description - no need to repeat the model definition as such, but enough to say `\Sigma_{i} = S_i^\top \Sigma_{g(i)} S_i`
where `g(i)` is mapping the subject `i` to its covariance group `g`.

In addition, 
3) discuss how the parameter vector `theta` contains all covariance parameters for the let's say total `G` covariance groups